purchase_discount/models/purchase_order.py

```diff
@@ -65,6 +65,8 @@ def _get_stock_move_price_unit(self):
         HACK: This is needed while https://github.com/odoo/odoo/pull/29983
         is not merged.
         """
+        if hasattr(self.env, "ocb"):
```

Comment:
naive question : I didn't know ocb adds an extra val in self.env. is it documented somewhere ?

Reply:
Check OCB#1193, which is flagged on the header :wink:

Reply:
thanks !